xdsl/dialects/builtin.py

```diff
@@ -510,6 +513,7 @@ def from_params(
 AnyVectorType: TypeAlias = VectorType[Attribute]
 
 _TensorTypeElems = TypeVar("_TensorTypeElems", bound=Attribute, covariant=True)
+_TensorTypeInitElems = TypeVar("_TensorTypeInitElems", bound=Attribute)
```

Comment:
Why Init? If no specific reason, could we rather specify that it is the invariant one?

Reply:
Yes, could we change this to `_TensorTypeInvElems` instead?